index.html

```diff
@@ -4,9 +4,71 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+      integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="style.css" />
     <title>Anwar Hussaini's Portfolio</title>
   </head>
   <body>
-    <h1 class="heading-primary">My Portfolio</h1>
+    <header class="main-header bg-color">
+      <div class="notification-bar">
+        <img src="Light.png" alt="Notification Bar" class="notification-img" />
+      </div>
+      <div class="container">
+        <nav class="main-nav">
+          <div class="logo-box"></div>
+          <div class="menus-box"><i class="fa-solid fa-bars menu icon"></i></div>
+        </nav>
+      </div>
+    </header>
+
+    <main>
+      <section class="section-hero bg-color">
+        <div class="container">
+          <h1 class="heading-primary">
+            <div class="hero-name">
+              <span>Hey There.</span>
+              <span>I'm Anwar</span>
+            </div>
+            <div class="hero-job">
+              <span>I'm a software</span>
+              <span>developer</span>
```

Comment:
- [ ] Please use the `p` tag instead of the `span` tag because this way will be `semantically` correct and the screen readers so kindly replace the `p` tag with the `span` tag :slightly_smiling_face: